compose/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  cardano-node:
+    image: inputoutput/cardano-node:latest
+    environment:
+      - NETWORK=${NETWORK}
+    volumes:
+      - ~/${NETWORK}/:/data
+      - ~/${NETWORK}/:/ipc
+    restart: on-failure
+
+  cardano-wallet:
+    image: inputoutput/cardano-wallet:latest
+    volumes:
+      - ~/${NETWORK}/:/config
```

Comment:
We are also going to need `/ipc`. I'd suggest to use a named volume for that (and for the rest as well, there's no reason to mount this on the host machine on an accessible location. It's better if users don't fiddle with that :)). 

```yaml

services:
    cardano-node:
        [...]
        volumes:
        - node-data:/data
        - node-ipc:/ipc
        - node-config:/config
        [...]

    cardano-wallet:
        [..]
        volumes:
        - node-config:/config
        - node-ipc:/ipc

volumes:
    node-data:
    node-ipc:
    node-config:
```

Reply:
I agree, however the thing is that for starting testnet one needs to also provide a `testnet-genesis.yaml` to the cardano-wallet... so I figured it would be more convenient to be able to put it into more accessible location. On the other hand that genesis.yaml file is not easy to get in the first place... (In particular I don't even know how to get it from cardano-node docker as it is "hidden" somwehre in the `/nix/store`) so maybe we could limit this exemplary guide just to mainnet...